application/account-management/WebApp/global.d.ts

```diff
@@ -16,6 +16,8 @@ export declare global {
     APPLICATION_VERSION: string;
     /* User locale */
     LOCALE: string;
+    /* Application Insights Connection String */
+    APP_INSIGHTS_CONNECTION_STRING?: string;
```

Comment:
Use APPLICATIONINSIGHTS_CONNECTION_STRING`, which is special in the .NET SDK: https://learn.microsoft.com/en-us/azure/azure-monitor/app/sdk-connection-string?tabs=dotnet5#environment-variable